save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/views/ProjectView.kt

```diff
@@ -782,14 +845,27 @@ class ProjectView : AbstractView<ProjectExecutionRouteProps, ProjectViewState>(f
                 errorLabel = "Failed to fetch latest execution"
                 errorMessage =
                         "Failed to fetch latest execution: [${response.status}] ${response.statusText}"
-                isErrorOpen = true
+                latestExecutionId = null
             }
         } else {
-            val latestExecutionId = response
+            val latestExecutionIdNew = response
                 .decodeFromJsonString<ExecutionDto>()
                 .id
+            setState {
+                latestExecutionId = latestExecutionIdNew
+            }
+        }
+    }
+
+    private fun switchToLatestExecution(latestExecutionId: Long?) {
+        latestExecutionId?.let {
             window.location.href = "${window.location}/history/execution/$latestExecutionId"
         }
+            ?: run {
+                setState {
+                    isErrorOpen = true
```

Comment:
At this point, if `latestExecutionId == null`, then state should contain data about network request for latest execution, that ended unsuccessfully, rigth? I think it might be confusing, because the user will click the button once more and see the same error, because we only attempt this request once, on page mount. Maybe set error text to something like `Cannot fetch latest execution, please refresh the page and try again`?

Reply:
Ok. Changed text in error